src/components/WorkflowSelector.js

```diff
@@ -45,7 +46,7 @@ const WorkflowSelector = ({
       style: { marginBottom: '1rem', height: 50, flex: 'none' },
       text: workflowName,
       onClick: () => {
-        Ajax().Metrics.captureEvent(Events.workspaceDataOpenWithWorkflow, extractWorkspaceDetails({ namespace, name }));
+        void Metrics().captureEvent(Events.workspaceDataOpenWithWorkflow, extractWorkspaceDetails({ namespace, name }));
```

Comment:
Why do you need the void here?

Reply:
there's a lint warning on this, "fire and forget" async calls like this common Metrics call should be marked with void prefix to note that it is intentionally not being awaited on.

Reply:
there's a useMetrics hook that avoids this awkwardness, but switching over to that is not part of this wave of cleanup to keep things narrow.